.github/workflows/ci-publish-cloud-helm-chart.yaml

```diff
@@ -0,0 +1,43 @@
+defaults:
+  run:
+    shell: bash
+env:
+  CHART_VERSION: "0.0.0"
+  GAR_REPO: "us-central1-docker.pkg.dev/liferay-artifact-registry/liferay-helm-chart"
+jobs:
+  publish-cloud-helm-chart:
+    name: Publish Cloud Helm Chart
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - id: calculate-chart-version
+        name: Calculate chart version
+        run: |
+          LATEST_CHART_TAG=$(git describe --tags --match="helm-chart-*" HEAD)
+          PUBLISH_VERSION=${LATEST_CHART_TAG#"helm-"}
+          echo "CHART_VERSION=${PUBLISH_VERSION}"
+          echo "CHART_VERSION=${PUBLISH_VERSION}" >> ${GITHUB_ENV}
```

Comment:
@rotty3000 and I are discussing a versioning strategy for these helm releases. For now, this script assumes a tag format of `helm-chart-<version>`